CDP4Reporting.Tests/ViewModels/ReportDesignerViewModelTextFixture.cs

```diff
@@ -110,26 +110,21 @@ public override object CreateDataObject()
 	        using CDP4Reporting.Parameters;
 	        using System.Collections.Generic;
 
-	        public class TestReportingParameters: IReportingParameters
+	        public class TestReportingParameters: ReportingParameters
 	        {
-		        public IEnumerable<IReportingParameter> CreateParameters(object dataObject)
+		        public override IEnumerable<IReportingParameter> CreateParameters(object dataObject)
 		        {
-			        List<IReportingParameter> paramsList = new List<IReportingParameter>();
-			        paramsList.Add(new ReportingParameter(""param1"", typeof(int), 0));
-			        paramsList.Add(new ReportingParameter(""param2"", typeof(string), string.Empty));
+                    List<IReportingParameter> paramsList = new List<IReportingParameter>();
```

Comment:
Wrong indentation for this block.

Reply:
This is part of a text string that was probably originally copied with tabs instead of spaces.
Should be fine now.